lib/typescript/render/providers/chatplugin/components/RichText/index.tsx

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import ReactMarkdown from 'react-markdown';
 
-import {Message} from 'model';
-
 import styles from './index.module.scss';
 
 type RichTextRenderProps = {
-  message: Message;
   text: string;
   fromContact: boolean;
   fallback: string;
   containsRichText: boolean;
 };
 
 export const RichText = (props: RichTextRenderProps) => {
-  const {message, text, fromContact} = props;
+  const {text, fromContact} = props;
   return (
-    <div className={fromContact ? styles.contactContent : styles.memberContent} id={`message-item-${message.id}`}>
+    <div className={fromContact ? styles.contactContent : styles.memberContent}>
```

Comment:
Is the cypress test still passing?

Reply:
This id was a [duplicate](https://github.com/airyhq/airy/blob/3aa1f97fbe5137319ca43b4b6c64574b76185d57/frontend/ui/src/pages/Inbox/Messenger/MessageList/index.tsx#L172)